src/Command/CheckRequirements.php

```diff
@@ -0,0 +1,103 @@
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Formatter\OutputFormatterInterface;
+use Symfony\Component\Console\Formatter\OutputFormatterStyle;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Checks if the host's environment meets Manala's requirements (Vagrant, Ansible, etc.).
+ *
+ * @author Xavier Roldo <xavier.roldo@elao.com>
+ */
+class CheckRequirements extends Command
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('check:requirements')
+            ->setDescription("Checks if your host's environment meets Manala's requirements (Vagrant, Ansible, etc.)")
+        ;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $this->configureFormatter($output->getFormatter());
+
+        $violationList = new RequirementViolationList();
+        $requirementChecker = new RequirementChecker(
+            new HandlerFactoryResolver(),
+            new RequirementViolationLabelBuilder()
+        );
+
+        $repository = new RequirementRepository();
+        foreach ($repository->getRequirements() as $requirement) {
+            $output->writeln('Checking '.$requirement->getName());
+            $requirementChecker->check($requirement, $violationList);
+        }
+
+        if (count($violationList) > 0) {
+            foreach ($violationList as $violation) {
+                $description = $this->getFormattedViolationDescription($violation);
+                $output->writeln($description);
+            }
+        }
+
+        if (!$violationList->containsRequiredViolations()) {
+            $output->writeln('Congratulations ! Everything seems OK.');
+            if ($violationList->containsRecommendedViolations()) {
+                $output->writeln('Yet, some recommendations have been emitted (see above).');
+            }
+        }
+    }
+
+    /**
+     * @param OutputFormatterInterface $formatter
+     */
+    private function configureFormatter(OutputFormatterInterface $formatter)
+    {
+        $errorStyle = new OutputFormatterStyle('red');
+        $formatter->setStyle('error', $errorStyle);
+
+        $warningStyle = new OutputFormatterStyle('yellow');
+        $formatter->setStyle('warning', $warningStyle);
+    }
+
+    /**
+     * @param RequirementViolation $violation
+     *
+     * @return string Formatted violation description displayed to user
+     */
+    private function getFormattedViolationDescription(RequirementViolation $violation)
+    {
+        $resultPattern = $violation->isRequired() ? '<error>%s</error>' : '<warning>%s</warning>';
+        $displayedText = $violation->getLabel();
+        $displayedText .= $violation->getHelp() ? ' '.$violation->getHelp() : '';
+
+        return sprintf($resultPattern, $displayedText);
+    }
```

Comment:
I would like to propose you [a small patch](https://github.com/manala/manala/commit/fb1d1e27d26593dfcd4c323c9e4675e7764a7ba8) which makes use of `SymfonyStyle` as output. I find it nice and we use it in other commands, what do you think?